compiler/noirc_evaluator/src/acir/call/intrinsics/slice_ops.rs

```diff
@@ -0,0 +1,579 @@
+use crate::acir::{AcirDynamicArray, AcirType, AcirValue};
+use crate::errors::RuntimeError;
+use crate::ssa::ir::{dfg::DataFlowGraph, instruction::Intrinsic, value::ValueId};
+use acvm::{AcirField, FieldElement};
+
+use super::Context;
+
+impl Context<'_> {
+    pub(super) fn convert_slice_intrinsic(
```

Comment:
Totally optional, but I'm not sure if having this function is better than just inlining it into the caller, avoiding this double match and the `_` match arm.

I know the intention is to keep slice stuff together, but there's still these variants on the call site, so...